package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@piwikpro/ngx-piwik-pro-project",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Issue: We set minimum node version to 18, this is a breaking change and we should bump major version like we did in other packages